src/pages/audio-rewards-page/components/modals/HCaptchaModal.tsx

```diff
@@ -23,15 +23,17 @@ export const HCaptchaModal = () => {
   const [isOpen, setOpen] = useModalState('HCaptcha')
   const dispatch = useDispatch()
 
-  // here, we can also instead use a prop e.g. onClose to handle re-opening the reward modal if we want to decouple rewards and hcaptcha
-  const [, setRewardModalOpen] = useModalState('ChallengeRewardsExplainer')
-
-  const handleClose = useCallback(() => {
-    if (isOpen) {
-      setRewardModalOpen(true)
-      setOpen(false)
-    }
-  }, [setRewardModalOpen, isOpen, setOpen])
+  const handleClose = useCallback(
+    (userInitiated = true) => {
+      if (isOpen) {
+        if (userInitiated) {
+          dispatch(setHCaptchaStatus({ status: HCaptchaStatus.USER_CLOSED }))
+        }
+        setOpen(false)
```

Comment:
super nit, but handleClose can only be called when isOpen is true right? so i think we can remove the isOpen branch